### PR TITLE
extend by Exists function

### PR DESCRIPTION
**Pull request by nicoandrewss**

### CODE REVIEW

File: service.go [hunk 1]
```diff
@@ -97,3 +97,24 @@ func (s *Service) Set(ctx context.Context, ttl time.Duration, key string, val in
...
+	const errMessage = "failed to get value from redis"
+
+	result, err := s.client.Exists(ctx, key).Uint64()
+	if err != nil {
+		return false, errors.Wrap(err, errMessage)
+	}
+
+	existsInRedis := result == 1
+	if !existsInRedis {
+		return false, errors.Wrap(ErrRedisKeyNotFound, errMessage)
```
Comment (MaxBreida) [inline service.go:116]: I don't think this should throw any more errors, it's enough to know that the given key does not exist

Comment (MaxBreida) [reply service.go:116]: if I were supposed to use this function and receive an error, I wouldn't want to check for this particular error every time just to ignore it afterwards

Comment (nicoandrewss) [reply service.go:116]: so just return the bool it is then

**Comment by MaxBreida** [comment]
LGTM

**Pull request merged**